2025-05-24 12:58:21,156 INFO    MainThread:87748 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2025-05-24 12:58:21,156 INFO    MainThread:87748 [wandb_setup.py:_flush():76] Configure stats pid to 87748
2025-05-24 12:58:21,156 INFO    MainThread:87748 [wandb_setup.py:_flush():76] Loading settings from /Users/amandaworker/.config/wandb/settings
2025-05-24 12:58:21,156 INFO    MainThread:87748 [wandb_setup.py:_flush():76] Loading settings from /Users/amandaworker/Documents/Udacity/Machine Learning Engineer/reproducible_workflow/genre_classification/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/starter/check_data/wandb/settings
2025-05-24 12:58:21,156 INFO    MainThread:87748 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'exercise_14', 'run_group': 'dev'}
2025-05-24 12:58:21,156 INFO    MainThread:87748 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-05-24 12:58:21,156 WARNING MainThread:87748 [wandb_setup.py:_flush():76] Could not save program above cwd: /opt/miniconda3/envs/mlflow-60000e2b97642ccc38c7a9ee503977f0abd54819/bin/pytest
2025-05-24 12:58:21,156 INFO    MainThread:87748 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program_abspath': '/opt/miniconda3/envs/mlflow-60000e2b97642ccc38c7a9ee503977f0abd54819/bin/pytest', 'program': '/opt/miniconda3/envs/mlflow-60000e2b97642ccc38c7a9ee503977f0abd54819/bin/pytest'}
2025-05-24 12:58:21,156 INFO    MainThread:87748 [wandb_init.py:_log_setup():526] Logging user logs to /Users/amandaworker/Documents/Udacity/Machine Learning Engineer/reproducible_workflow/genre_classification/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/starter/check_data/wandb/run-20250524_125821-5rergdqc/logs/debug.log
2025-05-24 12:58:21,157 INFO    MainThread:87748 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/amandaworker/Documents/Udacity/Machine Learning Engineer/reproducible_workflow/genre_classification/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/starter/check_data/wandb/run-20250524_125821-5rergdqc/logs/debug-internal.log
2025-05-24 12:58:21,157 INFO    MainThread:87748 [wandb_init.py:init():566] calling init triggers
2025-05-24 12:58:21,157 INFO    MainThread:87748 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2025-05-24 12:58:21,157 INFO    MainThread:87748 [wandb_init.py:init():616] starting backend
2025-05-24 12:58:21,157 INFO    MainThread:87748 [wandb_init.py:init():620] setting up manager
2025-05-24 12:58:21,158 INFO    MainThread:87748 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-05-24 12:58:21,162 INFO    MainThread:87748 [wandb_init.py:init():628] backend started and connected
2025-05-24 12:58:21,166 INFO    MainThread:87748 [wandb_init.py:init():720] updated telemetry
2025-05-24 12:58:21,201 INFO    MainThread:87748 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2025-05-24 12:58:21,972 INFO    MainThread:87748 [wandb_run.py:_on_init():2262] communicating current version
2025-05-24 12:58:22,438 INFO    MainThread:87748 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.19.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-05-24 12:58:22,438 INFO    MainThread:87748 [wandb_init.py:init():804] starting run threads in backend
2025-05-24 12:58:23,737 INFO    MainThread:87748 [wandb_run.py:_console_start():2241] atexit reg
2025-05-24 12:58:23,737 INFO    MainThread:87748 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2025-05-24 12:58:23,737 INFO    MainThread:87748 [wandb_run.py:_redirect():2161] Wrapping output streams.
2025-05-24 12:58:23,737 INFO    MainThread:87748 [wandb_run.py:_redirect():2186] Redirects installed.
2025-05-24 12:58:23,738 INFO    MainThread:87748 [wandb_init.py:init():847] run started, returning control to user process
2025-05-24 12:58:34,195 WARNING MsgRouterThr:87748 [router.py:message_loop():77] message_loop has been closed
